README.md

```diff
@@ -154,27 +154,6 @@ The documentation is available here: [documentation](https://pypsa-earth.readthe
 <!-- readme: collaborators,contributors -start -->
 <table>
 <tr>
-    <td align="center">
```

Comment:
This changes again... very weird; need for changing the CI.
I'll do.

Reply:
Ok! Once it'll be approved I squash this branch